### PR TITLE
Build app

### PR DESCRIPTION
**Pull request by coleanthony1990**
This branch has our organized components with our code to display a random soup in a category dependent on which hogwarts house you choose. this branch include our api calls as well. 
Something we had a challenge with was getting the links to hide from view once a house was chosen. Thankfully, Scott, figured out we needed to have the links in their own separate component.

- [ ]  Bug fix (fixes an issue)
- [x]  New feature (adds functionality)
- [x]  Refactor

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

- [x]  I tested my changes in the browser
- [ ]  I tested with Cypress

# Checklist:

- [x]  My code follows the style guidelines of this project
- [ ]  I have performed a self-review of my own code
- [ ]  I have commented my code, particularly in hard-to-understand areas
- [ ]  I have made corresponding changes to the documentation
- [x]  My changes generate no new bugs
- [ ]  I have added tests that prove my fix is effective or that my feature works
- [ ]  New and existing unit tests pass locally with my changes
- [x]  Is your code D.R.Y?
- [x]  Does it follow SRP?

### CODE REVIEW

**Review by ohClaire — APPROVED** [review]

**Review by scottdgaines — APPROVED** [review]
Thanks for the PR description, Cole. Next steps, DRY things up, and see if we maybe get some of the route paths dynamic?

**Review by ohClaire — COMMENTED** [review]
Everyone was present when this was written

File: src/components/SoupCard.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  house: string
+}
+
+type Recipe = {
+  title: string;
+  ingredients: string;
+  servings: string;
+  instructions: string;
+};
+
```
Comment (sertmer) [inline src/components/SoupCard.tsx:13]: You love to see it! Great work setting up some interfaces!

File: src/components/SoupCard.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+export default function SoupCard(props: CardProps) {
+  const [soups, setSoups] = useState<Recipe[]>([]);
+  const [randomSoup, setRandomSoup] = useState<Recipe>({
+    title: '',
+    ingredients: '',
+    servings: '',
+    instructions: '',
+  });
```
Comment (sertmer) [inline src/components/SoupCard.tsx:21]: and hooks too! Looks like you're learning a ton!

**Review by sertmer — COMMENTED** [review]
Great work so far! I don't have much to add other than it's so exciting to see yall take this on and find success!

**Pull request merged**